### PR TITLE
Do we need Issue Templates?

### PR DESCRIPTION
**Pull request by squell**
I think these work quite nicely on the sudo-rs repository, and they might help to make creating an issue less daunting for newcomers.

Some notes:
* I'm not married to these particular templates,  just came up with these on the fly.
* Perhaps also add labels based on which template is chosen?

Feel free to add commits "on top of this PR" without going through a rigorous PR process.

### CODE REVIEW

**Review by tdittr — APPROVED** [review]
Looks nice, I am just not sure if we need them. But if they make it easier for people to collaborate I am all for it! Maybe it could make sense to also have labels for the different issue types and auto-assign those?

File: .github/ISSUE_TEMPLATE/ask-a-question.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Ask a question
+about: Let someone help you in case things are unclear
+title: ''
+labels: ''
```
Comment (squell) [inline .github/ISSUE_TEMPLATE/ask-a-question.md:5]:
```suggestion
labels: 'question'
```

File: .github/ISSUE_TEMPLATE/collaboration-request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Collaboration request
+about: Propose something you want to work on
+title: ''
+labels: ''
```
Comment (squell) [inline .github/ISSUE_TEMPLATE/collaboration-request.md:5]:
```suggestion
labels: 'collab-offer'
```

File: .github/ISSUE_TEMPLATE/report-a-mistake.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Report a mistake
+about: Create a report to help us improve teach-rs
+title: ''
+labels: ''
```
Comment (squell) [inline .github/ISSUE_TEMPLATE/report-a-mistake.md:5]:
```suggestion
labels: 'bug'
```

File: .github/ISSUE_TEMPLATE/suggest-an-improvement.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+name: Suggest an improvement
+about: Suggest an idea for teach-rs
+title: ''
+labels: ''
```
Comment (squell) [inline .github/ISSUE_TEMPLATE/suggest-an-improvement.md:5]:
```suggestion
labels: 'enhancement'
```

**Review by squell — COMMENTED** [review]
What about these?

**Comment by squell** [comment]
For the person merging: please squash.

**Review by hdoordt — APPROVED** [review]
I like this idea!

**Pull request merged**